src/presets/node/runtime/cli.ts

```diff
@@ -13,13 +14,13 @@ async function cli() {
   debug("StatusCode", r.status);
   debug("StatusMessage", r.statusText);
   // @ts-ignore
-  for (const header of r.headers.entries()) {
+  for (const header of Object.entries(r.headers)) {
```

Comment:
Was this needed for fix?

Reply:
At least with my environment it gave this error: `TypeError: r.headers.entries is not a function or its return value is not iterable`